imap_processing/swe/l1a/swe_science.py

```diff
@@ -130,50 +118,80 @@ def swe_science(decom_data):
         raw_counts = np.frombuffer(byte_data, dtype=np.uint8)
 
         # Uncompress counts. Uncompressed data is a list of 1260
-        # where 1260 = 15 seconds x 12 energy steps x 7 CEMs
+        # where 1260 = 180 x 7 CEMs
         # Take the "raw_counts" indices/counts mapping from
         # decompression_table and then reshape the return
-        uncompress_data = np.take(decompression_table, raw_counts).reshape(15, 12, 7)
-        raw_counts = raw_counts.reshape(15, 12, 7)
+        uncompress_data = np.take(decompression_table, raw_counts).reshape(180, 7)
+        # Save raw counts data as well
+        raw_counts = raw_counts.reshape(180, 7)
 
         # Save data with its metadata field to attrs and DataArray of xarray.
-        science_array.append(uncompress_data)
-        raw_science_array.append(raw_counts)
+        # Save data as np.int64 to be complaint with ISTP' FILLVAL
+        science_array.append(uncompress_data.astype(np.int64))
+        raw_science_array.append(raw_counts.astype(np.int64))
         add_metadata_to_array(data_packet, metadata_arrays)
 
-    met_time = xr.DataArray(
+    epoch_time = xr.DataArray(
         metadata_arrays["SHCOARSE"],
-        name="met_time",
-        dims=["met_time"],
-        attrs=dict(
-            description="Mission elapsed time",
-            units="seconds since start of the mission",
-        ),
+        name="Epoch",
+        dims=["Epoch"],
```

Comment:
Should this be "EPOCH" in caps?

Reply:
I had the same question at one point :) After looking at Bryan's code and ISTP, it seems like ISTP likes it `Epoch`.